unity-game/Assets/Scripts/ElevatorScripts/ManualElevator.cs

```diff
@@ -0,0 +1,226 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class ManualElevator : MonoBehaviour {
```

Comment:
What do you mean by "Extend" it?
I'm not even sure what a class does for a script.
That being said, they need to be kept separate and different from each other, otherwise they mess up a lot. Like, they flat don't work if you use one script for both.